### PR TITLE
cloudformation-guard 3.0.1

### PR DESCRIPTION
**Pull request by joshfried-aws**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
👋 @joshfried-aws thanks for the PR, your base branch is outdated, you need to do the refresh it before creating the new PRs next time to avoid merge conflict. Thanks!

File: Formula/cloudformation-guard.rb
Comment (chenrui333) [inline Formula/cloudformation-guard.rb:1]: we also did the formuale sharding.

**Comment by joshfried-aws** [comment]
@chenrui333 Hi, i didn't notice that. Sorry about that and thanks for updating it for me!

**Comment by chenrui333** [comment]
yeah, no problem at all.

**Review by chenrui333 — APPROVED** [review]

**Review by p-linnane — APPROVED** [review]

**Pull request merged**